### PR TITLE
Create a pull request template with checklist

### PR DESCRIPTION
**Pull request by AnthonyClark**
This is to try smooth through the PR process for adding new icons by addressing some of the most common reasons we request changes.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+** Checklist **
+- [ ] This PR includes no more than 5 new icons
+- [ ] All icons include `width="28" height="24"` in their attributes
+- [ ] All icons have solid white backgrounds
```
Comment (AnotherJoSmith) [inline .github/PULL_REQUEST_TEMPLATE.md]: Shouldn't this be transparent?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+- [ ] All icons have a corresponding entry in `db/payment_icons.yml`
+- [ ] I am confident that all icons are clear and easy to read/understand
```
Comment (AnotherJoSmith) [inline .github/PULL_REQUEST_TEMPLATE.md]: I don't think we need this.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+** Checklist **
+- [ ] This PR includes no more than 5 new icons
```
Comment (AnotherJoSmith) [inline .github/PULL_REQUEST_TEMPLATE.md]: Why the limit?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (AnthonyClark) [inline .github/PULL_REQUEST_TEMPLATE.md]: Keeping with the solid white requirement in the contributions section

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (AnthonyClark) [inline .github/PULL_REQUEST_TEMPLATE.md]: This one was the biggest inspiration for me to add this, let's keep it and see how useful it is

**Pull request merged**